src/ScriptBuilder.Tests/ScriptBuilder.Tests.csproj

```diff
@@ -14,6 +14,7 @@
   </ItemGroup>
 
   <ItemGroup>
+    <PackageReference Include="Newtonsoft.Json" Version="[11.0.1, 12.0.0)" />
```

Comment:
Why are these additions necessary? Plus tests would not use the range but 11.0.1 directly

Reply:
Ah saw it now in the PR description. But still the test should not use the range

Reply:
Fixed to 11.0.1